setup/USER_MODIFY_config

```diff
@@ -0,0 +1,27 @@
+
+# REQUIRED Set this path to the parent directory where you cloned the geoips repo.
+# ie, $GEOIPS_PACKAGES_DIR/geoips/tests/integration_tests/base_test.sh
+export GEOIPS_PACKAGES_DIR=$HOME/geoips
+
+# OPTIONAL Set this path to where conda is installed
+# This is not required if you activated conda in your .bashrc
+eval "$($HOME/mambaforge/bin/conda shell.bash hook)"
```

Comment:
I guess "eval" is needed here to work? Probably not ideal, but likely fine?

Reply:
I wonder if that line should be commented out by default?

Reply:
It is the line that conda tells you to run if you tell it not to initialize the shell

Reply:
Yeah,  sure,  comment it out by default

Reply:
I think I'm going to approve this as a "hidden Mindy-only feature" for now. She uses this for her environment initialization. I've got my own methods, too.